docs/content/development/extensions-core/approximate-histograms.md

```diff
@@ -92,17 +96,140 @@ query.
 |`numBuckets`             |Number of output buckets for the resulting histogram. Bucket intervals are dynamic, based on the range of the underlying data. Use a post-aggregator to have finer control over the bucketing scheme|7|
 |`lowerLimit`/`upperLimit`|Restrict the approximation to the given range. The values outside this range will be aggregated into two centroids. Counts of values outside this range are still maintained. |-INF/+INF|
 
+## Fixed Buckets Histogram
+
```

Comment:
nit: I think it's worth calling out here that the fixed bucket implementation is recommended mostly for when you know what you are doing and why, and pointing to datasketches for general purpose histogramming needs.

Reply:
Added a comment on that